cmd/chisel/cmd_cut_test.go

```diff
@@ -317,6 +315,47 @@ var cutTests = []cutTest{{
 		"/other-dir/":     "dir 0755",
 		"/other-dir/file": "symlink ../dir/file",
 	},
+}, {
+	summary: "Copyright is automatically extracted if exists",
+	pkgs: map[string][]byte{
+		"test-package": testutil.MustMakeDeb(
+			append(testutil.TestPackageEntries,
+				testutil.Dir(0755, "./usr/"),
+				testutil.Dir(0755, "./usr/share/"),
+				testutil.Dir(0755, "./usr/share/doc/"),
+				testutil.Dir(0755, "./usr/share/doc/test-package/"),
+				testutil.Reg(0644, "./usr/share/doc/test-package/copyright", "copyright"),
+			),
+		),
+		"other-package": testutil.PackageData["other-package"],
+	},
+	release: map[string]string{
+		"slices/mydir/test-package.yaml": `
+			package: test-package
+			slices:
+				myslice:
+					contents:
+						/dir/file:
+		`,
+		"slices/mydir/other-package.yaml": `
```

Comment:
This package is not used, please remove it.

Reply:
Done.